### DISCUSSION TITLE
How to automatically add copied content to relevant tab?

### DISCUSSION THREAD

**Discussion by careb0t** [hluk/CopyQ#2920] [Q&A]
Hi! I've been using CopyQ as my main clipboard manager on NixOS for the past couple of weeks and it is the best clipboard manager I have ever used. I am trying to learn about how tabs work, and I was wondering if the following is possible. 

Basically I would like to have 3 tabs, one for copied text, one for copied images, and one for copied screenshots from Flameshot. Right now I only have a single tab with all copied content inside. Would it be possible to make CopyQ detect if the copied content is an image or text, then whether or not a copied image is a Flameshot screenshot, and then place the copied content in the correct tab?

**Comment by hluk** [comment]
There is [Image Tab](https://github.com/hluk/copyq-commands/tree/master/Automatic#image-tab) command that moves images to a separate tab instead of the default one (see [README.md](https://github.com/hluk/copyq-commands/blob/master/README.md) for details on how to add the command).

It may be possible to distinguish the clipboard content further (depending on what formats Flameshot provides). You can check the content and formats in system clipboard from the main window with Ctrl+Shift+C shortcut.
